src/components/event/EventItem.jsx

```diff
@@ -16,30 +16,37 @@ function Event ({ event, onLike }) {
     <Card className={classes}>
       <Card.Body>
         <Row>
-          <Col xs={8}>
+          <Col>
             <Link to={`/event/${event.id}`}>
               <h3>{event.title}</h3>
             </Link>
+            {event.organizers &&
+              <p>
+                {event.organizers.map(organizer => (
+                  <span key={organizer.id} className='mr-3'>{organizer.website_url ? <a href={organizer.website_url}>{organizer.title}</a> : organizer.title}</span>
```

Comment:
I was actually thinking about an internal link here (`/organizer/${organizer.id}`) for consistency with stuff we show in the event list.